src/test/kotlin/test/de/itemis/mps/gradle/JBRDownloadTest.kt

```diff
@@ -140,7 +140,11 @@ class JBRDownloadTest {
                 .withPluginClasspath(cp)
                 .build()
         Assert.assertEquals(TaskOutcome.SUCCESS, result.task(":downloadJbr")?.outcome)
-        Assert.assertTrue(File(testProjectDir.root, "build/jbrDownload").exists())
+        val jbrDownloadDir = File(testProjectDir.root, "build/jbrDownload")
+        Assert.assertTrue(jbrDownloadDir.exists())
+        val jbrReleaseFile = File(jbrDownloadDir, "jbr/release")
```

Comment:
What is that release directory? I never saw it in any of the donwloaded JBR distros.

Reply:
Just cross-checked, it seems like it is only available on the linux version of the JBR

Reply:
That file is packaged with all platform distros, I was running the tests locally on Windows, which worked. But probably you checked it on OSX - there the JBR directories layout is simply different and therefore the file would not be found in the test. If you wish, you could fix it. But I think, we are good enough with it for now.